sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2247,10 +2294,36 @@ def hpat_pandas_series_add_number_impl(self, other, level=None, fill_value=None,
 @sdc_overload_method(SeriesType, 'sub')
 def hpat_pandas_series_sub(self, other, level=None, fill_value=None, axis=0):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.sub
+
+    Examples
```

Comment:
Limitation section stated that `fill_value` is not supported

Reply:
Now the not implement parameters are described in the notes section. Need to remove them to limitations?